data_registry/process_manager/task/coverage.py

```diff
@@ -0,0 +1,70 @@
+import gzip
+import shutil
+import tempfile
+from pathlib import Path
+
+import ocdscardinal
+
+from data_registry import models
+from data_registry.process_manager.util import TaskManager, skip_if_not_started
+from exporter.util import Export
+
+
+def get_keys_for_building_block(coverage, building_block):
+    return [key for key in coverage if key.endswith(building_block)]
+
+
+class Coverage(TaskManager):
+    final_output = False
```

Comment:
I added a commit to explain this attribute better.

```suggestion
    final_output = True
```

Reply:
Hmm, should we change the attribute name too? "Whether the task produces a final output, like a bulk download." is False for this case, but because of what the attribute is used for, we have to set it to True 🤔

Reply:
I consider writing to the database as an "output". What would be clearer?